docs/product/explore/session-replay/index.mdx

```diff
@@ -1,41 +1,12 @@
 ---
 title: "Session Replay"
 sidebar_order: 70
-description: "Use Session Replay to get video-like reproductions of user interactions to improve your website or app experience."
+description: "Use Session Replay to get reproductions of user sessions to improve your app experience."
 ---
 
-<Include name="feature-stage-beta-session-replay.mdx" />
+Sentry offers Session Replay for Web and Mobile. While Web support has been available and stable, Mobile Replay has recently been available in Beta.
```

Comment:
@lizokm you might want to adjust this paragraph. We can also do it in follow up PRs so no rush